WSL/wsl1-wsl2.md

```diff
@@ -0,0 +1,52 @@
+---
+title: An overview on the the Windows Subsystem for Linux
+description: Learn about the Windows Subsystem for Linux, the different versions and ways you can use them.
+keywords: BashOnWindows, bash, wsl, windows, windowssubsystem, gnu, linux
+ms.date: 03/23/2020
+ms.topic: article
+---
+
+# WSL1 vs. WSL2
+
+
+WSL 2 uses the latest and greatest in virtualization technology to run its Linux kernel inside of a lightweight utility virtual machine (VM). However, WSL 2 will NOT be a traditional VM experience. 
+
+## Benefits and Best Practices 
+
+Feature | WSL1 | WSL2
+--- | --- | ---
+ Integration between Windows and Linux| ✅|✅
+ Fast boot times| ✅ | ✅
+ Small resource foot print| ✅ |✅
+ Managed VM| ❌ | ✅
+ Full Linux Kernel| ❌ |✅
+ Full system call compatibility| ❌ | ✅
+ Runs with VM Ware and VirtualBox| ✅ |❌
+ Fastest cross OS file performance| ✅ | ❌
+
+
+
+
+With full system call compatibility and lightning fast speeds, WSL2 can be used for all of your typical Windows and Linux operations. However, its best to use WSL1 for the following cases.
```

Comment:
```suggestion
In general, we recommend that you use WSL 2 since it offers faster performance and 100% system call compatibility. However, there are some scenarios where you might prefer using WSL 1. 
```